txpool/txpool.go

```diff
@@ -321,6 +323,16 @@ func (p *TxPool) SetSigner(s signer) {
 	p.signer = s
 }
 
+// SetSealing sets the sealing flag
+func (p *TxPool) SetSealing(sealing bool) {
+	p.sealing.Store(sealing)
+}
+
+// Sealing returns the current set sealing flag
+func (p *TxPool) Sealing() bool {
```

Comment:
You can un-export this method

Reply:
Fixed 52735acfc16c2ce6849fcc18334ad8c4c7180a0e